src/Day1.hs

```diff
@@ -65,7 +65,7 @@ classic :: Classic State
 classic = Classic
   { title = "Day 1"
   , rows = screenH
-  , cols = screenW
+  , cols = screenW + hudW
```

Comment:
I mean that `cols` should only be `screenW`. Not plus anything else

Reply:
Oh okay got it.